lib/tmuxinator/cli.rb

```diff
@@ -226,6 +227,44 @@ def show_continuation_prompt
       def kill_project(project)
         Kernel.exec(project.kill)
       end
+
+      def tmux_session_exists
+        ENV["TMUX"] ? true : false
+      end
+
+      def should_append?(options)
+        options[:append] == true ||
+          (options[:append] != false && Tmuxinator::Config.options[:append])
```

Comment:
Could you walk me through the use case for the right side of this or?

Reply:
the idea is that can set append in the `Tmuxinator::Config.options` so all sessions are appending by default.
if append is not provided by the `options` (case `!= false`) we check that it is set in the global Config.